framer/SVGPath.coffee

```diff
@@ -30,6 +30,31 @@ class exports.SVGPath extends SVGBaseLayer
 			@_path = @_svg.querySelector(link)
 		@_length = @_path.getTotalLength()
 
+
+	updateStroke: ->
+		startLength = @strokeStart ? 0
+		endLength = @strokeEnd ? @length
```

Comment:
I think we need to make sure strokeEnd > length is handled like an angle > 360. Otherwise animations that overshoot have strange effects.

Reply:
I've tested this, and what we actually want to do is what we do now, limiting strokeEnd to path.length, otherwise you get the weird effects you describe